README.md

```diff
@@ -28,31 +28,33 @@ Advanced Configuration
 ----------------------
 This script supports several flags, all of which are optional.
 
-* `--archlinux_mirror`  
+* `--archlinux_mirror`
```

Comment:
Leave the two trailing spaces; this instructs Markdown to start the argument description on the next line instead of the same line.